src/components/Book/index.js

```diff
@@ -91,12 +95,22 @@ const Book = ({item}) => {
                 </Card.Text>
             </Card.Body>
             <div className="card-book-selectors">
-                <div onClick={() => addBook(item)}>
```

Comment:
Can you put the onClick bit back in :)

Reply:
Ooops! Must have accidentally taken it out when mergin - sorry!